docs/new_instrument.rst

```diff
@@ -763,6 +788,27 @@ Note that :py:mod:`pysat` assumes that this flag is True if no variable is
 present. Thus, specifying only ``_test_download = {'': {'Level_1': False}}``
 has the same effect, and Level 2 tests will still be run.
 
+Load Options
```

Comment:
@jklenzing I think I included the missing test options.  Can you verify that?

Reply:
this looks good to me.